### PR TITLE
Add has phenotype or disease and has disease

### PR DESCRIPTION
**Pull request by laurenechan**
Proposed 'has disease' relation in reference to issue #478 
@matentzn I look forward to your review and continued iteration on this proposed change.

### CODE REVIEW

File: src/ontology/ro-edit.owl [hunk 1]
```diff
@@ -6127,6 +6131,21 @@ AnnotationAssertion(rdfs:label obo:RO_0015003 "subcluster of")
 SubObjectPropertyOf(obo:RO_0015003 obo:BFO_0000050)
 TransitiveObjectProperty(obo:RO_0015003)
 
+# Object Property: obo:RO_0016001 (disease or disorder)
```
Comment (matentzn) [inline src/ontology/ro-edit.owl:6134]: not a relation name.. shouldnt the name be something like `has phenotype or disease` or some such?

File: src/ontology/ro-edit.owl [hunk 2]
```diff
@@ -6127,6 +6131,21 @@ AnnotationAssertion(rdfs:label obo:RO_0015003 "subcluster of")
...
+AnnotationAssertion(dce:date obo:RO_0016001 "2021-11-05T17:30:14Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:RO_0016001 "disease or disorder")
+
+# Object Property: obo:RO_0016002 (has disease)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0016002 "A relationship that holds between an organism and a disease. Here a disease is construed broadly as a disposition to undergo pathological processes that exists in an organism because of one or more disorders in that organism.")
+AnnotationAssertion(dce:contributor obo:RO_0016002 "http://orcid.org/0000-0002-7463-6306")
+AnnotationAssertion(dce:date obo:RO_0016002 "2021-11-05T17:30:44Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:RO_0016002 "has disease")
+AnnotationAssertion(rdfs:seeAlso obo:RO_0016002 "https://github.com/oborel/obo-relations/issues/478#issuecomment-895384341")
```
Comment (matentzn) [inline src/ontology/ro-edit.owl:6146]: just link the issue, not the comment

File: src/ontology/ro-edit.owl [hunk 3]
```diff
@@ -6127,6 +6131,21 @@ AnnotationAssertion(rdfs:label obo:RO_0015003 "subcluster of")
 SubObjectPropertyOf(obo:RO_0015003 obo:BFO_0000050)
 TransitiveObjectProperty(obo:RO_0015003)
 
+# Object Property: obo:RO_0016001 (disease or disorder)
+
+AnnotationAssertion(dce:contributor obo:RO_0016001 <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(dce:date obo:RO_0016001 "2021-11-05T17:30:14Z"^^xsd:dateTime)
```
Comment (matentzn) [inline src/ontology/ro-edit.owl:6137]: link the github issue here as well

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/ro-edit.owl (hunk 1 above)
Comment (laurenechan) [reply src/ontology/ro-edit.owl:6134]: Sorry, I think I was a little confused at your instructions then. Should I not have made a parent class for 'has disease' and 'has phenotype'? Should they just both be siblings under the top class?

Comment (matentzn) [reply src/ontology/ro-edit.owl:6134]: They are not classes, :) no I think just rename this object property to "has phenotype or disease" or whatever Chris suggested in the ticket is fine. But they are not classes!!

Comment (laurenechan) [reply src/ontology/ro-edit.owl]: Ok, not classes!! I relabeled the object property and added a note not to use this class directly. I think that's appropriate here, correct? @matentzn

**Review by matentzn — APPROVED** [review]
I like this but try to get some approval from @balhoff @cmungall or @dosumis

**Review by cmungall — APPROVED** [review]

**Comment by lschriml** [comment]
Reviewed, approved.

**Pull request merged**